lib/tools/system-calls.js

```diff
@@ -411,6 +412,21 @@ systemCallMethods.launchAVD = async function (avdName, avdArgs, language, countr
   return proc;
 };
 
+systemCallMethods.getAdbVersion = _.memoize(async function () {
+  try {
+    let adbVersion = (await this.adbExec('version'))
+      .replace(/Android\sDebug\sBridge\sversion\s([\d\.]*)[\s\w\-]*/, "$1");
+    return {
+      versionString: adbVersion,
+      versionFloat: parseFloat(adbVersion).toFixed(1)
```

Comment:
@mykola-mokhnach i'll change the `parseFloat` and remove `toFixed` since it's not getting the last numbers